FlightReservation/src/screens/SignUp.js

```diff
@@ -52,12 +70,12 @@ const SignUp = function ({navigation}) {
               <Text style={style.title}>
                 <Span text="signup" />
               </Text>
-              <View style={style.textField__text}>
+              <View>
                 <TextField
                   {...formProps}
                   label="Name"
```

Comment:
You can use the internationalization that you have already implemented here.

Reply:
Done